R/get_genbank.R

```diff
@@ -7,14 +7,14 @@
 #' @export
 #'
 #' @examples
-get_genbank <- function(url = "https://ftp.ncbi.nlm.nih.gov/genomes/Viruses/AllNuclMetadata/AllNuclMetadata.csv.gz", location = here::here("./Source/")){
+get_genbank <- function(gb_url = "https://ftp.ncbi.nlm.nih.gov/genomes/Viruses/AllNuclMetadata/AllNuclMetadata.csv.gz", location = here::here("Source/AllNuclMetadata.csv.gz")){
   
   message("this takes a minute or two")
   # download with wget
-  system(paste0("wget ", url, " -qP ", location))
+  system(paste0("wget ", gb_url, " -qO ", location))
```

Comment:
this tell wget to write the file in a location vs using the prefix. When the prefix is used wget does not overwrite older versions of the file.